### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mzouros**

### CODE REVIEW

**Review by spyroukostas — CHANGES_REQUESTED** [review]
Remove duplicate vocabulary type enum

File: eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/ServiceController.java [hunk 1]
```diff
@@ -199,7 +199,6 @@ public List<String> getChildrenFromParent(@RequestParam String type, @RequestPar
     @ApiImplicitParams({
             @ApiImplicitParam(name = "ids", value = "Comma-separated list of Resource ids", dataType = "string", paramType = "path")
     })
-    @PreAuthorize("hasRole('ROLE_ADMIN') or hasRole('ROLE_EPOT')")
```
Comment (spyroukostas) [inline eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/ServiceController.java:202]: By removing auth you are exposing sensitive info. Did you take care of this issue by updating the aspect that removes sensitive user information?

File: eic-registry/src/main/java/eu/einfracentral/config/security/AuthoritiesUpdaterAspect.java [hunk 2]
```diff
@@ -22,20 +22,20 @@ public AuthoritiesUpdaterAspect(AuthoritiesMapper authoritiesMapper) {
...
-    public void updateAuthorities() {
-        logger.trace("Updating Provider Roles");
-        // update provider roles
-        try {
-            authoritiesMapper.updateAuthorities();
-        } catch (RuntimeException e) {
-            logger.error("Could not update authorities", e);
-        }
-    }
+//    @AfterReturning(pointcut = "execution(* eu.einfracentral.registry.manager.ProviderManager.add(..)) " +
```
Comment (spyroukostas) [inline eic-registry/src/main/java/eu/einfracentral/config/security/AuthoritiesUpdaterAspect.java:25]: Remove commented out code.

**Review by spyroukostas — CHANGES_REQUESTED** [review]

File: eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/ServiceController.java (hunk 1 above)
Comment (mzouros) [reply eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/ServiceController.java:202]: Sure

Comment (mzouros) [reply eic-registry-rest/src/main/java/eu/einfracentral/registry/controller/ServiceController.java:202]: Sure

**Review by spyroukostas — APPROVED** [review]

**Pull request merged**